src/main/java/uk/gov/hmcts/reform/unspec/launchdarkly/FeatureToggleService.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.unspec.launchdarkly;
+
+import com.launchdarkly.sdk.LDUser;
+import com.launchdarkly.sdk.server.interfaces.LDClientInterface;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Slf4j
+@Service
+public class FeatureToggleService {
+
+    public static final LDUser UNSPEC_SERVICE_USER = new LDUser.Builder("civil-unspec-service")
```

Comment:
Should we move that key to config?

Reply:
I don't see that to be overridden in any of the environments.